converter.c

```diff
@@ -20,7 +20,42 @@ char* base10to32enc(long unsigned int value) {
 			return NULL;
 	} while (value /= OUTPUT_BASE);
 	while (sizeof(buffer) / sizeof(buffer[0]) - offset <= WORD_SIZE_IN_BASE )/* Align to word size */
-		buffer[--offset] = '0';
-	copyStr(&tmp, buffer + offset);
-	return tmp; /* warning: this must be freed by the user */
+//		buffer[--offset] = '0';
+//	copyStr(&tmp, buffer + offset);
+//	return tmp; /* warning: this must be freed by the user */
+//}*/
+
+char * base10to2Wierd(long unsigned int value) {
+	int i;
+	const char base2[3] = "./";
+	char *numInBaseWeird2 = (char *) malloc(WORD_SIZE-1);
+	if (!numInBaseWeird2) {
+		return NULL;
+	}
+	for (i = 0; i < WORD_SIZE-1; i++) {
+		numInBaseWeird2[i] = '.';
+	}
+	i = WORD_SIZE-1;
+	while(value!=0)
+	{
+		numInBaseWeird2[i] = base2[value%2];
+		value=value/2;
+		i--;
+	}
+	return numInBaseWeird2;
 }
+
+char *base10Decimal(long unsigned int value){
```

Comment:
function name need to explain what the function does

Reply:
function does this: put into a string the value of the memory line.
can be done more easily like this:
 if(value>999){ sprintf( destStr, "0%d", value )}
else{sprintf( destStr, "%d", value }

Reply:
change function to use sprintf
done.